static/app/components/replays/table/replayTableColumns.tsx

```diff
@@ -571,9 +615,18 @@ export const ReplaySlowestTransactionColumn: ReplayTableColumn = {
   },
 };
 
+const DetailsLink = styled(Link)`
+  z-index: 1;
+  margin: -${p => p.theme.space.md};
+  padding: ${p => p.theme.space.md};
+  line-height: 0;
+`;
```

Comment:
This is here to give this new column a little extra invisible clickable space. 
The "View replay details" link is just an icon after all, so it's easy to miss it and click the row instead (which will load up the snippet)